test.js

```diff
@@ -151,13 +152,15 @@ test('removeTrailingSlash and removeDirectoryIndex options)', t => {
 	};
 	t.is(normalizeUrl('http://sindresorhus.com/path/', options1), 'http://sindresorhus.com/path');
 	t.is(normalizeUrl('http://sindresorhus.com/path/index.html', options1), 'http://sindresorhus.com/path');
+	t.is(normalizeUrl('http://sindresorhus.com/#/path/', options1), 'http://sindresorhus.com/#/path');
```

Comment:
I don't think that the `removeTrailingSlash` should remove a trailing slash in this case.

`removeTrailingSlash` was added with intent to remove the trailing splash from the pathname.

The test should be instead:

```js
t.is(normalizeUrl('http://sindresorhus.com/foo/#/bar/', options1), 'http://sindresorhus.com/foo#/bar');

```

Lets wait for @sindresorhus feedback first, though.

Reply:
> I don't think that the removeTrailingSlash should remove a trailing slash in this case.

👍 

> `t.is(normalizeUrl('http://sindresorhus.com/foo/#/bar/', options1), 'http://sindresorhus.com/foo#/bar');`

This proposed test is not correct. The slash after `bar/` should not be removed.

Reply:
> This proposed test is not correct. The slash after bar/ should not be removed.

Oops. Copy-paste failure.